### PR TITLE
Reorder contributing steps

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Reorder the steps in the Contribute code or content section.

### CODE REVIEW

File: docs/contributing/02-contributing.md [hunk 1]
```diff
@@ -69,16 +69,15 @@ To contribute code or content to a given Kyma repository, follow these steps:
...
-11. Wait for the Kyma maintainers to review and approve your PR. The maintainers can approve it, request enhancements to your change, or reject it.
-
-> **NOTE:** The reviewer must check if all related CI tests have completed successfully before approving the PR.
+> **NOTE:** For more information on code owners' responsibilities, see the [Kyma working model](../governance/01-governance.md) document.
 
-12. When the maintainers approve your change, merge the PR or wait until the Kyma Bot merges it for you.
+12.  Wait for the Kyma maintainers to review and approve your PR. The maintainers can approve it, request enhancements to your change, or reject it.
 
-13. As a PR owner, you are responsible for ensuring the success of the submission process, including pre- and postsubmit Prow jobs. You can ask for more help from the Prow jobs owner only after the initial analysis.
+> **NOTE:** The reviewer must check if all related CI tests have completed successfully before approving the PR.
```
Comment (NHingerl) [inline docs/contributing/02-contributing.md:78]:
```suggestion
   > **NOTE:** The reviewer must check if all related CI tests have completed successfully before approving the PR.
```

File: docs/contributing/02-contributing.md [hunk 2]
```diff
@@ -69,16 +69,15 @@ To contribute code or content to a given Kyma repository, follow these steps:
...
 
-> **NOTE:** For more information on code owners' responsibilities, see the [Kyma working model](../governance/01-governance.md) document.
+13.  . When the maintainers approve your change, merge the PR or wait until the Kyma Bot merges it for you.
```
Comment (NHingerl) [inline docs/contributing/02-contributing.md:80]:
```suggestion
13.  When the maintainers approve your change, merge the PR or wait until the Kyma Bot merges it for you.
```

File: docs/contributing/02-contributing.md [hunk 3]
```diff
@@ -69,16 +69,15 @@ To contribute code or content to a given Kyma repository, follow these steps:
...
+11.  As a PR owner, you are responsible for ensuring the success of the submission process, including pre- and postsubmit Prow jobs. You can ask for more help from the Prow jobs owner only after the initial analysis.
 
-11. Wait for the Kyma maintainers to review and approve your PR. The maintainers can approve it, request enhancements to your change, or reject it.
-
-> **NOTE:** The reviewer must check if all related CI tests have completed successfully before approving the PR.
+> **NOTE:** For more information on code owners' responsibilities, see the [Kyma working model](../governance/01-governance.md) document.
 
-12. When the maintainers approve your change, merge the PR or wait until the Kyma Bot merges it for you.
+12.  Wait for the Kyma maintainers to review and approve your PR. The maintainers can approve it, request enhancements to your change, or reject it.
```
Comment (NHingerl) [inline docs/contributing/02-contributing.md:76]:
```suggestion
12. Wait for the Kyma maintainers to review and approve your PR. The maintainers can approve it, request enhancements to your change, or reject it.
```

**Review by NHingerl — COMMENTED** [review]
just wrestling with the punctuation :D

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**